docs/notebooks/2.3 Dust Soiling with Kimber.ipynb

```diff
@@ -0,0 +1,407 @@
oNjgqF/mN5ZD/xED7eHdsPDz+d23JETAKz4DLFAzN7/7HtgqburqlsgkSxZIVHIT/vyYFs9mMfFMkWQJM0NPrnQoct1kMMxwLfFFttbg8pJRtHQ+C0+sEya4R/0i+joZTEpA2GeWQIUU1Qy3fCIs/HRG+y8E/rx+DwAOm8KclvTaZJuGYhHJDjOpmGTu/sc7k9YJpZ9EIik6iq7r0DHHHMMxxxxj/H3ssceycOFCbrnlFr75zbEXoy984QvccMMNxt8HDx5k0aJFeYm1KFEUIds+8udgr8vaZvX74myY4RqlQ0l0wCgaFAWmnC8eAMP7YMdvRcIr34mW1lNhzZ+EAaPsNmQew7vhXwtECdG7evK779kfgGkXC0NeScaowRNg7ESLgkVRUDUtMomw/z5xHphwEiz6XM7jjKDlBPGIQjJn8WTzRZqmGQoV+8BrsOMBqFkEM8d6Qhhdh9QSSbIf/DcEXKC6c7YLXYWU7KW3UPJ8uaBjwFN8iRaz0DQIjEhF63jGdRDan4SpFwhFjUQiSRlTEy1NTU1YrVba29sjnm9vb6e1NTnzP7vdzsqVK9mxI7obttPpxOl0Gn8PDAykH/B4wlGf1c0ZHi19r8Fbd0LrGVC/LK1tOXQz3FJStIzGUQdLvwEWE76i9mqYdWX+9ysZhUV8D7OY8EwJe7V4SDJGH7vGSzRbLaAGQklpQChKDj+U9fNxuiQzCNdLM5I1YVW1kIrH1v8GbP62SDhHSbSUnEfLiu/A1Avjf8e3fF8kY+Z+DMqaUt5FSNESn4jPTQ3Aq58SsbWcKFQVRUp4uZs+SVNUaBr4g2oGW1X+FIZ9b8AjJ8CM9wpTdqlsHH/s/gu88UXYcxac/JDZ0UgkRYmpVx2Hw8GqVat47LHHjOdUVeWxxx6LUK3EIxAI8OabbzJx4sRchTl+8PSAtzcnmzY8CjqfE4av3RvS3lbJlQ51bYAdvwNvX+g5ew0s/UrWPHIkRUjNXKFkOT+9biNpI1UsWSfU3jn2OsZANzybMfEMoS6b86EcRjcKvwu2/lx0nBlDYnVEqsOx8PO4vX6xaCU+6R1R1w0pWkok0aIo0LQGahfEXmfLzfDm18CTXge65D1awmh/XJivPvMuoWwsYjxh3Qn1e4fiQoO7asTDm0dl44H7RYLH2y2TLGbTuxE2fBA23ZTf/TobRQfMaZfkd78SSQlheunQDTfcwBVXXMGRRx7J0UcfzU9+8hOGh4e56qqrAHj/+9/P5MmTufnmmwH4xje+wZo1a5gzZw59fX18//vfZ+/evXzwgx80898oDbb9HDbfDEu/Cou/mNVNG6VDdfOh4X1QPSftbRlmuKXS3nnbz2HP36H3jdh1+vnitc9C42qYfC5YnYnXl5QW7g64fwZMOBVOuMsc88USJJEZbviyCLVG7SLxyCcH7odXrodtv4B3vB0xyAq1qY79f0RtExwHPWliURQsLcfChNjt5C2lpmhJhlkfEG190+1IlORbFfG5VUwWyT1HQ9FfB9y+gNkhFCdLvgqtp8mSkULAtR92/gEajoIlX87ffud8CGZfA5r8Dkkk6WJ6ouXSSy+ls7OTr371q7S1tbFixQr++9//Gga5+/btwxJmkNfb28s111xDW1sb9fX1rFq1ivXr10vflWzQ8yqoHqjIfocBwwx30plwxNUZbavkFC2Na6D3dZj5vtBzmioURpCWXDwtBrbBWz8QDvcXHIRy2TZ93HH4EVGX726XSZYskoyqIJhDwPQcgq1K3NBPWhfHVCZ7u/MHz+M2q5KwI0zJKFpG2uCxU2DG5WJSI15pTpZaeyfbbUcDkdw7+pas7Nds3L7QfUJxHjcKXDoifrXkMemlKNAcO+kpySM182H5t6DMBOW+ooimDBKJJC0K4tvz8Y9/nI9//ONRlz355JMRf//4xz/mxz/+cR6iGoesvU/U5VZnv8NJQE08o5ssDpvYRskkWuZ/HOZ9LPI5dwfcOxFQ4PI8/Z+2Klj0BSFRL/Iky+6uYXZ0DHHk9HrqKx1mh5MeI23w2ueEGeHRv8nPPme+BxpW5ayEcLyiJ5rjKUF0/5GI0iF3J4wcFB4eVTNyGGEYU84VD3XsLGYyw9RE3W1Go3tt2a2JXxfyaCnyc//e22DgLWGGm+MZ6mS9coxuUSXmhuvxh45jfzHeMyiKTHqPd6rnZF1lHpeAV0z+NR4ly8YkkgwpxoJVSa5QFKhfkROpqKFosWR+0jZKhwIldEOoKKMuaCZc3ComwYpvw+rf5X/fWea+1w6y6WA/z+3sMjuU9PENwJ6/wt7b87vf2gXQnJxHliQ59Il0a5zzn644iJh03/MPeGilaPWeb6J0HAuVDsXBKEFJ7vysdxyyWSxCHv8PBZ48N+q6tqC61V/s5/7Z18CaP+dl8JS0R4vuEbT/Phjen9ug8kh4ifG4KjlLF2+/OOds/nbR+/NI0uTwQ/DwaqG6k0gkGSETLRIxc6nldqbHp7fv3PELuLsBdv4p7W2F2jsX4exUOCNtonVetPe+rAUuC4iHJCXUsJvp/pEivlEsa4aVP4BleTbAk2QdI9GcROlQIDxBYa+C8kn56zrU/iSo/piLNcMMN4seLfq1IQVFi1rsqgt7Fcx6v1AOJeK+qXC7A/o2pbWrUNeh+O+vogC+fnj10/DADHF9KgHCzXCLs3QIeP3/xMM3lPt97btLqBn2/F2WjBQKAY9QOXvyZIY8vBes5ULdKpFIMkKeRSVw4D549QZY8ClY8Mmc7EIv87FrLlGWoLrT3lbJeLTs+pNonTftEjj+jshlikJeVS1774CaBVC/PH/7zBHD3tBAMdUyhoLCUQ8LP52ffakBeHgNtKwVpQwF0k64VNCTf/ESFHpZZXiikNlXi0c+6H0DHjsZKqfDudvBYs9oc8kOaX2qXjpkgemXw+TzwBK93K9kPFpSQfUKZUGakyHJKosUgMAIWsNqqOiH8ta09ldoeAPhiZYivWfY8l3xc8GnRZIul0y/RPi0
```

Comment:
biweekly is one word (consistent with how it is written in the previous paragraph)